src/js/extension.js

```diff
@@ -4,6 +4,16 @@ var Extension;
 
     /* global Util */
 
+    var passThroughHelpers = [
```

Comment:
Could you add a short description of the goal of this variable?
Also, it could be helpful to add a description on the `forEach` bellow to explain what is done